cards/abdominal-pain-diagnostics/card.md

```diff
@@ -10,6 +10,7 @@ categories:
     - Surgery
     - EBM
 drugs: null
+pubmed: (Diagnosis/Broad[filter]) AND (Imaging and laboratory testing in acute abdominal pain.) 
```

Comment:
Ah, ok that makes sense. Thought for some reason you wanted to add it somewhere on the rendered card on the site which is what had me confused.

Carry on, good sir!